src/xenia/kernel/xthread.h

```diff
@@ -88,7 +88,8 @@ struct X_KTHREAD {
   char unk_10[0xAC];             // 0x10
   uint8_t suspend_count;         // 0xBC
   uint8_t unk_BD;                // 0xBD
-  uint16_t unk_BE;               // 0xBE
+  uint8_t unk_BE;                // 0xBE
+  uint8_t core_id;               // 0xBF
```

Comment:
What's the source of the `core_id` name? It maybe a bit confusing that it's different than `current_cpu` if that's not the original name of the variable (Windows NT uses the word "Processor" for similar purposes also).